packages/braid-design-system/src/lib/components/List/List.tsx

```diff
@@ -108,6 +108,10 @@ export const List = ({
     size: sizeProp,
     tone: toneProp,
   });
+
+  const defaultSpace = size === 'xsmall' ? 'xsmall' : 'small';
```

Comment:
Should also reduce the default for `small` text:

```suggestion
  const defaultSpace =
    size === 'xsmall' || size === 'small' ? 'xsmall' : 'small';
```

Reply:
ahh, will update